apps/fluent-tester/src/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -207,44 +209,29 @@ const StandardCallout: React.FunctionComponent = () => {
     setScrollviewContents((arr) => [...arr, 1]);
   }, [setScrollviewContents]);
 
+  const onOpenCalloutOnHoverAnchor = (_e: InteractionEvent, value: boolean) => {
```

Comment:
We should use React.useCallback for these functions. Otherwise they get recreated on every render.

Reply:
Comment applies to a lot of the other changes as well

Reply:
Yeah, in some places I used callbacks, but in some I missed it